### PR TITLE
Invalid playbookID in incident type

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24193
 
## Description
Add a validation that checks the playbookID field in incidents type is not UUID.

## Screenshots
Screen Shot 2020-06-04 at 14 51 09

 
## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/tests/incident_type_test.py [hunk 1]
```diff
@@ -84,3 +84,18 @@ def test_is_changed_from_version(current_from_version, old_from_version, answer)
     structure.current_file = current_from_version
     validator = IncidentTypeValidator(structure)
     assert validator.is_changed_from_version() is answer
+
+
+IS_VALID_PLAYBOOK_ID = [
+    ('valid playbook', True),
```
Comment (teizenman) [inline demisto_sdk/commands/common/tests/incident_type_test.py:90]: Add another test with an empty string as playbookId

Comment (barchen1) [reply demisto_sdk/commands/common/tests/incident_type_test.py:90]: done

File: demisto_sdk/commands/common/hook_validations/incident_type.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
+import re
 from distutils.version import LooseVersion
 
 from demisto_sdk.commands.common.errors import Errors
 from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
     ContentEntityValidator
 
+INVALID_PLAYBOOK_ID = r'[\w\d]{8}-[\w\d]{4}-[\w\d]{4}-[\w\d]{4}-[\w\d]{12}'
```
[the remark this answers is not in the document]
Comment (barchen1) [reply demisto_sdk/commands/common/hook_validations/incident_type.py:9]: done
